src/tests/algorithms_test/calorimetry_HEXPLIT.cc

```diff
@@ -88,7 +88,7 @@ TEST_CASE( "the subcell-splitting algorithm runs", "[HEXPLIT]" ) {
     Esum+=subcell.getEnergy();
     i++;
     if (i%12==0){
-      REQUIRE(abs(Esum-E[i/12-1])/E[i/12-1]<tol);
+      REQUIRE(fabs(Esum-E[i/12-1])/E[i/12-1]<tol);
```

Comment:
```suggestion
      REQUIRE(std::abs(Esum-E[i/12-1])/E[i/12-1]<tol);
```
We can use the c++ function

Reply:
can -> should